### PR TITLE
fix: set longer timeout on the longer release pipeline

### PR DESCRIPTION
**Pull request by ralphbean**
For a release with 20 images, this takes over an hour currently.

### CODE REVIEW

**Comment by johnbieren** [comment]
This doesn't seem to work. When I tried it, I got
`admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "timeout"`

**Comment by ralphbean** [comment]
Looks like `timeout` may be deprecated. I updated to use `timeouts`: https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#configuring-a-failure-timeout

File: pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ metadata:
 spec:
   description: >-
     Tekton pipeline to release content to registry.redhat.io registry
+  timeouts:
```
Comment (johnbieren) [inline pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml:14]: I believe these are valid on a `kind: PipelineRun`, not `kind: Pipeline`

Comment (johnbieren) [reply pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml:14]: You can set it per task in this yaml though like this https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#configuring-the-failure-timeout

Comment (ralphbean) [reply pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml:14]: Hm, I'm afraid it won't work as we expect if we set it on the task.

Will the 1h timeout on the pipelinerun ignore our 2h timeout on the task, if we set it on the task?

Comment (ralphbean) [reply pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml]: Updated in c1553f0 to try it.

**Review by johnbieren — APPROVED** [review]

File: pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml (hunk 1 above)
Comment (johnbieren) [reply pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml]: I'm not sure

**Pull request merged**